packages/core/src/indexing/sqlite/sqlite-index-api.ts

```diff
@@ -26,22 +26,39 @@ export class SqliteIndexApi implements DatabaseIndexApi {
     this.insertionOrder = new InsertionOrder(dbConnection)
   }
 
+  public getActiveModelsToIndex(): Array<StreamID> {
+    // Helper function to get array of active models that are currently being indexed by node
+    // TODO: update to runtime check once adminAPI unlocks to load models on the fly
```

Comment:
Even when we have the adminAPI and the table tracking the set of indexed models, I don't think we should be doing a database query here every time a stream is updated.  Instead I think the this class should maintain an in-memory cache of which streams are indexed and what their corresponding tables are.  This cache can get populated from the database at startup, and updated anytime there is a call to the admin API endpoint to add or remove a model from the set of models to index.

Reply:
yeah, I like that approach. the idea was to have a starting point here as I think we will start out still with in-memory cache of models as per defined in the config file. I'd change the name then to `getActiveModelsToIndexFromCache` as the helper function and the subsequent cache updates as you described it then are done via the future extension.

Reply:
nitpick, but IMO should leave the name as `getActiveModelsToIndex`, the fact that the data is coming from a cache is an implementation detail.  I think for now the `getActiveModelsToIndex` function can just return the models as listed in the config file, and then later can be from the cache that's build off the database table